lib/nsq-cluster/nsqd.rb

```diff
@@ -10,17 +10,20 @@ class Nsqd < ProcessWrapper
 
 
   def initialize(opts = {})
-    @host = opts[:host] || '127.0.0.1'
-    @tcp_port = opts[:tcp_port] || 4150
-    @http_port = opts[:http_port] || 4151
-    @lookupd = opts[:nsqlookupd] || []
-    @msg_timeout = opts[:msg_timeout] || '60s'
```

Comment:
hi @DanielleSucher ! happy to see you pop up in my work email! :wave: :love_letter: I haven't worked on this project so I'll let Robby answer your actual question.